docker-compose.yml

```diff
@@ -12,11 +12,11 @@ services:
             PGSQL_DB_1_POSTGIS: "true"
         volumes:
             # Use a volume to enable database persistence across restarts
-            - /tmp/nodewatcher-db:/var/lib/postgresql/9.5/main
+            - ~/nodewatcher-db:/var/lib/postgresql/9.5/main
```

Comment:
OK, but read my proposal for the better solution as well and consider making a change to fix this so that in the future you do not have to commit in temporary changes for your development to work.

Also, why do you have to commit this in? You can have local changes to files which you do not commit. git allows partial commits. Maybe use some GUI like GitHub client which allows an easy way to select what you want to commit.